lib/enigma.rb

```diff
@@ -1,8 +1,14 @@
+require "./lib/encryptor"
+require "./lib/decryptor"
+require "./lib/code_breaker"
+
 class Enigma
 
-  attr_reader :alphabet
+  attr_reader :encryptor, :decryptor, :code_breaker
   def initialize
-    @alphabet = ("a".."z").to_a << " "
+    @encryptor = Encryptor.new
+    @decryptor = Decryptor.new
+    @code_breaker = CodeBreaker.new
```

Comment:
Instantiates each class and used them to fulfill the functionality of Enigma. This allows me to encapsulate the different behaviors.